assignments/callbacks.js

```diff
@@ -1,34 +1,93 @@
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
+function getFirstElement(array){      // function for firstItem
+  return array[0];
+}
+
 function firstItem(arr, cb) {
-  // firstItem passes the first item of the given array to the callback function.
+  return cb(arr);
```

Comment:
There seems to be a bit of confusion on this page. You should be passing the results to the callback function.